setup/sql/dbupdate_04.php

```diff
@@ -24790,4 +24790,14 @@ function writeCtrlClassEntry(ilPluginSlot $slot, array $plugin_data)
     }
     $setting->set('sty_media_cont_mig', 2);
 }
+?>
+<#5428>
+<?php
+global $DIC;
+$DIC->database()->modifyTableColumn("usr_data", "login", [
```

Comment:
@sragfw We MUST NOT add additional database update steps to `dbupdate_04.php`. Please add this schema change to the corresponding hotfix file for ILIA 5.4.x

Reply:
Right, thanks for your hint